buckets/redis/bucket_factory.go

```diff
@@ -294,8 +250,8 @@ func newConfigAttributes(cfg *pbconfig.BucketConfig, idle string, dyn bool) *con
 		defaultBucket}
 }
 
-func toRedisKey(namespace, bucketName, suffix string) string {
-	return namespace + ":" + bucketName + ":" + suffix
+func toRedisKey(version int32, namespace, bucketName, suffix string) string {
+	return strconv.Itoa(int(version)) + ":" + namespace + ":" + bucketName + ":" + suffix
```

Comment:
Minor nit, are we better off formatting key as `namespace:buckenName:suffix:version`, for greater readability when manually looking through contents in Redis?

Reply:
Yeah I agree with that.